java/src/org/openqa/selenium/remote/http/ClientConfig.java

```diff
@@ -64,6 +64,16 @@ public static ClientConfig defaultConfig() {
       null);
   }
 
+  public static ClientConfig configWithTimeouts(int connectTimeoutSeconds, int readTimeoutSeconds) {
```

Comment:
This also would not be needed because the current builder lets you set these values `      ClientConfig.defaultConfig().readTimeout(Duration.ofSeconds(1))
              .connectionTimeout(Duration.ofSeconds(1));  `